examples/experimental/dagster-airlift/dagster_airlift/in_airflow/base_proxy_operator.py

```diff
@@ -22,6 +39,16 @@
 logger = logging.getLogger(__name__)
 
 
+class ExecutableAssetIdentifier(TypedDict):
```

Comment:
Don't love creating a new concept here but it's purely internal.

Reply:
fully support